saleor/graphql/translations/mutations.py

```diff
@@ -60,6 +63,22 @@
 }
 
 
+def validate_input_against_model(model: Model, input_data: dict):
+    errors = {}
+    for field_name, value in input_data.items():
+        model_field = model._meta.get_field(field_name)
+        if not isinstance(model_field, CharField):
+            continue
+        if value and len(value) > model_field.max_length:
```

Comment:
Hmm, would that work if we called `model.full_clean()` here? We do it in `BaseMutation.clean_instance` for instance.

Reply:
I've updated implementation to use that approach.